src/email/email.service.ts

```diff
@@ -10,7 +10,9 @@ export class EmailService {
   ) {}
   async sendEmail(data: any, tempPassword: string) {
     const { personalEmail } = data;
-    const message = `Hi ${data.firstName} ${data.lastName},
+    const message = `
```

Comment:
@vision-jm 
gawin natin itong default message for now pero mag-add ng message parameter para sa ibang custom messages/texts